deployment/v3/utils/prop_migrator/file_updater.py

```diff
@@ -0,0 +1,296 @@
+import csv
+import os
+import sys
+
+different_properties_csv = 'different_properties.csv'
+value_takes_priority_csv = 'knowledge/1.1.5.5-value-takes-priority.csv'
+lts_value_takes_priority_csv = 'knowledge/lts-Value-takes-priority.csv'
+new_directory = 'new-config/v1.2.0.1-B1'
+manual_adjudication_csv = 'manual-adjudication.csv'
+file1_only_csv = 'file1_only.csv'
+intentionally_removed_csv = 'knowledge/intentionally-removed-in-lts.csv'
+file2_only_csv = 'file2_only.csv'
+decent_default_value_csv = 'knowledge/new-property-with-decent-default-value.csv'
+log_file = 'logs.txt'
+
+matching_combinations = []
+non_matching_combinations = []
+matching_rows = []
+updated_property_files = []
+similar_new_value = []
+unresolved_combinations = set()  # Use a set to avoid duplicates
+case2_matching_combinations = []
+case2_unresolved_combinations = set()  # Use a set to avoid duplicates
+case3_matching_combinations = []
+case3_unresolved_combinations = set()  # Use a set to avoid duplicates
+existing_combinations = set()
+case2_existing_combinations = set()
+case3_existing_combinations = set()
+key_values = set()
+
+# Read different_properties.csv
+with open(different_properties_csv, 'r') as file:
+    reader = csv.DictReader(file)
+    properties_dict = {(row['New Property File'], row['Key']): row['Old Value'] for row in reader}
+
+# Read 1.1.5.5-value-takes-priority.csv
+with open(value_takes_priority_csv, 'r') as file:
+    reader = csv.DictReader(file)
+    for row in reader:
+        property_file_name = row['Property file name']
+        key = row['key']
+
+        # Compare the combinations and add to the corresponding list
+        if (property_file_name, key) in properties_dict:
+            matching_combinations.append((property_file_name, key))
+        else:
+            non_matching_combinations.append((property_file_name, key))
+
+# Check if the property files exist in the new directory and update their values
+for combination in matching_combinations:
+    property_file_name, key = combination
+    new_property_file_path = os.path.join(new_directory, property_file_name)
+
+    if os.path.isfile(new_property_file_path):
+        old_value = properties_dict[combination]
+
+        # Update the value in the property file
```

Comment:
check if mentioned file is present in latest config dir. If not present display the error as property file not fould. log the same and exit.